src/broker/broker.c

```diff
@@ -281,6 +282,26 @@ static int setup_profiling (const char *program, int rank)
     return (0);
 }
 
+static int increase_rlimits (void)
+{
+    struct rlimit rlim;
+
+    /*  Increase number of open files to max to prevent potential failures
+     *   due to file descriptor exhaustion (e.g. zuuid_new() failure to
+     *   open /dev/urandom
```

Comment:
nit: end parentheses :P

Reply:
Thanks! Good catch. I pushed a fixup just in time. :-)